prow/kube/prowjob.go

```diff
@@ -79,6 +79,8 @@ type ProwJobStatus struct {
 	URL             string       `json:"url,omitempty"`
 	PodName         string       `json:"pod_name,omitempty"`
 	BuildID         string       `json:"build_id,omitempty"`
+	UUID            string       `json:"uuid,omitempty"`
```

Comment:
I added a separate field here so that the job name and UUID do not have to remain the same.